it/repositories/UploadJourneyRepositoryISpec.scala

```diff
@@ -197,15 +199,19 @@ class UploadJourneyRepositoryISpec extends IntegrationSpecCommonBase {
       await(repository.updateStateOfFileUpload("1234", callbackModel2, isInitiateCall = true))
       await(repository.getNumberOfDocumentsForJourneyId("1234")) shouldBe 2
       await(repository.removeFileForJourney("1234", "ref1"))
-      await(repository.getNumberOfDocumentsForJourneyId("1234")) shouldBe 1
+      eventually {
+        await(repository.getNumberOfDocumentsForJourneyId("1234")) shouldBe 1
+      }
       await(repository.getUploadsForJourney(Some("1234"))).get.head shouldBe callbackModel2
```

Comment:
Should this line be in the eventually block?

Reply:
It's what is failing the test....2 does not equal 1